### DISCUSSION TITLE
Death Notification

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#388] [Q&A] [answered]
I failed to update the launch.bat so i clone the repository again but my death notification seems not to work, since the players just died and i didn't got the notice.

I did disable some patches though, here they are:
Patch disabled: Grablobby,InfiniteScanRange.Jeb,NoCooldown,OneHanded,Shovel,TeleportWithItems.

**Comment by winstxnhdw** [comment]
That shouldn’t happen. What does your logs say?

**Reply by Dexteriority** [reply]
I had no error logs though :(

**Reply by winstxnhdw** [reply]
Then I have no clue either. Could be a mod causing problems. Try doing `fakedeath` and see if the notif appears.

**Reply by Dexteriority** [reply]
I will be looking through mods and trying the fakedeath

**Reply by Dexteriority** [reply]
I used a vanilla and used the fakedeath, but the notification is not there?
Log is placed in the Lethal folder right? i have no files named with lc-hax

**Reply by winstxnhdw** [reply]
@xAstroBoy I believe this was your feature. Any ideas?

**Reply by winstxnhdw** [reply]
Just to confirm, your DeathNotificationPatch is there right?

**Reply by Dexteriority** [reply]
Yes sir its there!
![image](https://github.com/winstxnhdw/lc-hax/assets/157716300/ab6b80b7-4644-4c0e-a5d7-808a0811c067)

**Comment by Dexteriority** [comment]
@winstxnhdw i found out that its caused if i inject the lc-hax after the ship landed.
I tried to inject from lobby, it works fine.
Same issue has been found where ESP won't read the Enemies and Entrance/Exit if injected after the ship landed.

**Reply by winstxnhdw** [reply]
Good catch. DeathNotifcation not working when you inject after ship has land still doesn’t make sense though.

**Reply by winstxnhdw** [reply]
![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/de7a1ef4-d5be-4fbb-b4e9-cfdd8449a8d6)

I am in pain. I can replicate your issue and I can confirm that the method used to initialise the ESPs are also being activated. But for some reason it's not working.

**Reply by winstxnhdw** [reply, marked as answer]
Ok, I solved the issue for this one. I'll push soon. The issue was a race condition.

**Reply by Dexteriority** [reply]
I see. Your hardwork are praiseable my guy!! Great work there!
